NEWS.md

```diff
@@ -1,14 +1,14 @@
 # ModernDive 0.4.0
 
-Some highlights:
+## Highlights
 
 1. [`infer` package](http://infer.netlify.com/) is ready for prime-time! Thus we made a first pass at incorporating it into the book in Chapters 9 and 10 on confidence intervals and hypothesis testing!
```

Comment:
@rudeboybert Chapter 11 also includes `infer` for simple linear regression. Did that get merged in?

Reply:
Oops, no! Please modify before merging!